test/e2e/run-test.sh

```diff
@@ -21,13 +21,28 @@ set -x # so users can see what is going on
 env="kind"
 test="bash"
 fail_flag=""
-
+setup_flags=""
+existing_wec_contexts=""
 
 while [ $# != 0 ]; do
     case "$1" in
-        (-h|--help) echo "$0 usage: (--released | --env | --test-type | --kubestellar-controller-manager-verbosity \$num | --transport-controller-verbosity \$num | --fail-fast)*"
+        (-h|--help) echo "$0 usage: (--released | --env (kind|ocp) | --test-type (bash|ginkgo) | --kubestellar-controller-manager-verbosity \$num | --transport-controller-verbosity \$num | --fail-fast | --existing-wec-contexts \$context1,\$context2,... | --scenario \$scenario_name)*"
+                    echo "where:"
+                    echo "  \$num: integer verbosity level"
+                    echo "  \$context1,\$context2,...: comma-separated list of existing WEC contexts"
+                    echo "  \$scenario_name: scenario to use (default, wds-on-host, its-on-host, combined)"
```

Comment:
Like the other enumerated types, this enumeration could be folded into the main syntax expression.